tests/storage/datavolume.go

```diff
@@ -239,16 +274,15 @@ var _ = SIGDescribe("DataVolume Integration", func() {
 		It("Check disk expansion accounts for actual usable size", func() {
```

Comment:
An idea for the test:
- Use guestless VM with empty PVC (there's no need for a real VM in this test as far as I can tell)
- Check "real" usable size on PVC prior to creating a VM
- Delete dummy pod, Create VM
- Assert disk.img virtual size is less than size in step 2

The extra pod may cost a few seconds but the test will consume less resources
Should be more straightforward to test this way

Reply:
wdyt?

Reply:
Just updated the test with your recommendation, definitely less convoluted than calculating the available space after importing. Let me know what you think.

Reply:
Just tried with empty PVC and the disk size is handled in https://github.com/kubevirt/kubevirt/blob/v1.4.0/pkg/host-disk/host-disk.go#L266, which ends up being bigger than the one in `PossibleGuestSize` so it skips expansion and thus this PR's logic.

That said, the current test passes locally, probably because after aligning the size free size to 1Mib it ignores the few bytes that the blank image occupies.